pkg/webhooks/router/admission.go

```diff
@@ -19,11 +19,16 @@ package router
 import (
 	"fmt"
 	"net/http"
+	"strings"
 	"sync"
+
+	"volcano.sh/volcano/cmd/webhook-manager/app/options"
 )
 
 type AdmissionHandler func(w http.ResponseWriter, r *http.Request)
 
+var defaultEnabledAdmission = []string{"/jobs/mutate", "/jobs/validate", "/podgroups/mutate", "/pods/validate", "/queues/mutate", "/queues/validate"}
```

Comment:
there are 7 admission by default https://github.com/volcano-sh/volcano/blob/68429f4fa89995119ea4e54239b76b82839e0440/cmd/webhook-manager/main.go#L33-L39

BTW, suggust move `admit_pod.go` and `admit_pod_test.go` into pods/validate dir.

Reply:
done